subt/zmq-subt-x2.json

```diff
@@ -121,6 +124,7 @@
               ["rosmsg.depth", "detector.depth"],
 
               ["localization.pose3d", "app.pose3d"],
+              ["fromrospy.pose3d", "app.pose3d"],
```

Comment:
```suggestion
              ["offseter.pose3d", "app.pose3d"],
```

Reply:
I am not sure about this change - discussion is now over e-mail.

Reply:
I am withdrawing the suggestion.